docs/decisions/0000-url-format.md

```diff
@@ -0,0 +1,34 @@
+# URL Patterns for Parsley Application
+
+* status: accepted 
+* date: 2023-02-22
+* authors: Mohamed Khelif
+
+## Context and Problem Statement
+
+The Parsley application is a web-based tool used for viewing and analyzing logs files from both Evergreen and user uploaded logs. The application has several pages, and each page has its unique URL pattern.
+
+<!-- This is an optional element. Feel free to remove. -->
```

Comment:
I think you should delete comments and headings you're not using.

Reply:
Done.